src/ocm/components/clusterConfiguration/operators/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  OPERATOR_NAME_LVM,
+  Cluster,
+  FeatureSupportLevelData,
+  ExposedOperatorName,
+  OPERATOR_NAME_LVMS,
+} from '../../../../common';
+
+export const handleLVMS = ({
```

Comment:
The name is not clear, I would rename it to something like

`getActualLVMOperatorName`